docs/sdk/cosmos-sdk-cli.md

```diff
@@ -0,0 +1,34 @@
+# cosmos-sdk-cli 
+Create a new blockchain project based on cosmos-sdk with a single command.
+
+---
+
+# Installation
+
+```shell
+$go get github.com/cosmos/cosmos-sdk
+$cd $GOPATH/src/github.com/cosmos/cosmos-sdk
+$make install_cosmos-sdk-cli
+```
+
+This will install a binary cosmos-sdk-cli
+
+# Creating a new project
+
+**$cosmos-sdk-cli init** _Your-Project-Name_
+
+This will initialize a project, the dependencies, directory structures with the specified project name.
+
+### Example:
+```shell
+$cosmos-sdk-cli init testzone -p github.com/your_user_name/testzone
+```
+-p remote project path (optional). If this is not provided, it creates testzone under $GOPATH/src/
```

Comment:
can you put the `-p [remote project path]` in backticks? (This is how you format inline code in markdown)

Reply:
Sorry, I meant single backticks, not triple backticks, thats what makes it inline